securedrop/journalist_templates/col.html

```diff
@@ -62,6 +64,18 @@
         {% endfor %}
       </ul>
 
+      <!-- Confirmation modal for selected documents -->
+      <div id="delete-selected-confirmation-modal" class="modal-dialog">
+        <a href="#close" class="external"></a>
+        <div>
+          <a href="#close" title="Close" class="close">X</a>
+          <h2>{{ gettext('Delete Confirmation') }}</h2>
+          <p>{{ gettext('Are you sure you want to delete the selected documents?') }}</p>
+          <a href="#close" id="cancel-selected-deletions" title="Cancel" class="btn sd-button">Cancel</a>
+          <button type="submit" name="action" value="confirm_delete" id="delete-selected" class="sd-button danger">Delete</button>
```

Comment:
We should make sure each string is wrapped in `gettext` for translators, so instead of `'Delete'`, we should use `{{ gettext('Delete') }}`. This is also true for `Cancel`.